scripts/lib/api/mergeClassMembers.ts

```diff
@@ -94,12 +89,7 @@ export async function mergeClassMembers<T extends HtmlToMdResultWithUrl>(
     }
   }
 
-  // remove merged results
-  const finalResults = reject(results, (result) =>
-    includes(["method", "attribute", "property"], result.meta.apiType),
-  );
```

Comment:
This is where the pages were being deleted because we were assuming certain `apiType`s were merged. Now we just check if they're in `mergedMemberUrls`.

Reply:
Exactly.